org.fdroid.Repomaker.json

```diff
@@ -0,0 +1,263 @@
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-frameworks-base/android-libandroidfw_7.0.0+r33-1_amd64.deb",
+                    "sha256": "948cc995b217d2607adfbacd7ee45a9e777e29eb364766c9f1d3d57799276291"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-system-core/android-liblog_7.0.0+r33-1_amd64.deb",
+                    "sha256": "29a0c1178de6267876e9cadc2a676bab3a0d8b59ea178e0fb25eff0f617dd70c"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-system-core/android-libcutils_7.0.0+r33-1_amd64.deb",
+                    "sha256": "46ce10de01130b1d35e4359cbfac7cc7780b7bababe7cb950419346f40ebbb87"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-system-core/android-libbacktrace_7.0.0+r33-1_amd64.deb",
+                    "sha256": "a67befa0bef158f00b3b5fe340da43af4ee625690b70da302186fd1c1deb8cfb"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-system-core/android-libziparchive_7.0.0+r33-1_amd64.deb",
+                    "sha256": "ced85bae70a1f51fc911cb365a9c71009006c180e503b738345953716666058b"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-system-core/android-libbase_7.0.0+r33-1_amd64.deb",
+                    "sha256": "4bb6a3900d423f9b567bc7b9139c9c908edadd011f36b917880664ee7463243a"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/a/android-platform-external-libunwind/android-libunwind_7.0.0+r1-4_amd64.deb",
+                    "sha256": "d5891de063a4a67c890f56311528629942d7bb63113c638b54ce2d67e0fe588f"
+                },
+                {
+                    "type": "file",
+                    "url": "http://ftp.debian.org/debian/pool/main/p/p7zip/p7zip-full_16.02+dfsg-3+deb9u1_amd64.deb",
+                    "sha256": "a782560e02f99986e7b388bed4a437b61822d0288cfdeb1954189d206498b7e9"
+                }
+            ]
+        },
+        "python-deps.json",
+        "fdroidserver-deps.json",
+        {
+            "name": "fdroidserver",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python setup.py sdist",
+                "mv dist/fdroidserver-0.8.tar.gz .",
+                "pip3 install --no-index --find-links=\"file://${PWD}\" --prefix=${FLATPAK_DEST} fdroidserver"
```

Comment:
I believe this can be simplified to remove the dist step, just do `pip3 install --prefix=${FLATPAK_DEST} .`